quickannotator/client/src/components/fileDropUploader/fileDropUploader.tsx

```diff
@@ -169,6 +170,9 @@ const FileDropUploader = (props: any) => {
                             <p>
                                 Supported Annotation files:.json, .geojson
                             </p>
+                            <p>
+                                Supported Bunch Slides and Annotations import:.tsv
```

Comment:
Replace with "Supported tabular formats for bulk import of slides and annotations: .tsv"

Should also not hardcode this ideally. Can we get these from the const values?